main.js

```diff
@@ -73,14 +74,45 @@ function displayPalette(event) {
     currentPalette.replaceUnlockedColors();
   }
 
-  for (i = 0; i < currentPalette.colors.length; i++) {
+  for(i = 0; i < currentPalette.colors.length; i++) {
+    if(currentPalette.colors[i].locked === false) {
     paletteContainer.innerHTML +=
     `<article class="color-container" data-id="${currentPalette.colors[i].id}">
       <div class="color-box color1" style="background:${currentPalette.colors[i].hexcode};"></div>
       <span>
         <p class="hexcode">${currentPalette.colors[i].hexcode}</p>
-        <i class="lock">🔓</i>
+        <i class="fa-solid fa-lock-open" data-id="${currentPalette.colors[i].id}"></i>
       </span>
     </article>`;
+  } if(currentPalette.colors[i].locked === true) {
```

Comment:
could we move this next if down to its own line, instead of having it share a line with the previous if's closing curly brace?

Reply:
actually could this be an else?